### PR TITLE
Adding minor parameter for edit function

### PR DESCRIPTION
**Pull request by ValentinBrclz**

### CODE REVIEW

File: lib/bot.js [hunk 1]
```diff
@@ -430,7 +430,7 @@ module.exports = (function() {
 			});
 		},
 
-		edit: function(title, content, summary, callback) {
+		edit: function(title, content, summary, minor, callback) {
```
Comment (macbre) [inline lib/bot.js]: Let's avoid changing the parameters order. Please support the lack of `minor` argument by checking if it's a function.

Comment (ValentinBrclz) [inline lib/bot.js]: I carefully read the given lines of getUser() and the [example](https://github.com/ValentinBrclz/nodemw/blob/devel/examples/getUsers.js), but I am not sure how you see this... Could you be more specific on how you would implement it?

Comment (macbre) [inline lib/bot.js]:
```js
if(typeof minor === 'function') {
	callback = minor;
	minor = undefined;
}
```

**Comment by ValentinBrclz** [comment]
Ping @macbre :shipit:

**Comment by macbre** [comment]
@ValentinBrclz, sure - thanks for applying my suggestions. And sorry for the delay :)

**Comment by macbre** [comment]
And published as v0.8.2

**Comment by ValentinBrclz** [comment]
:raised_hands: Thank you, no worries for the delay

**Pull request merged**